api/graphql/tests/test_reservations/test_reservation_queries.py

```diff
@@ -1252,6 +1252,45 @@ def test_order_by_created_at(self):
         assert_that(content.get("errors")).is_none()
         self.assertMatchSnapshot(content)
 
+    def test_order_by_order_status(self):
+        res1 = ReservationFactory(name="this should be 1st")
+        PaymentOrderFactory(status=PaymentStatus.CANCELLED, reservation=res1)
```

Comment:
Should these enums be also OrderStatus? For concistency? or does that mean a larger refactor?

Reply:
I second the motion 👍

Reply:
I very much approve the suggestion! Added a new commit with fancy find-and-replace changes 👍